ui/src/common/containers/__tests__/CitationSummaryGraphContainer.test.jsx

```diff
@@ -146,8 +161,15 @@ describe('CitationSummaryGraphContainer', () => {
         },
       }),
     });
-    const wrapper = mount(<CitationSummaryGraphContainer store={store} />);
-    wrapper.find(CitationSummaryGraph).prop('onSelectBarChange')(null);
+    const wrapper = mount(
+      <Provider store={store}>
+        <CitationSummaryGraphContainer />
+      </Provider>
+    );
+    const onSelectBarChange = wrapper
+      .find(CitationSummaryGraph)
+      .prop('onSelectBarChange');
+    onSelectBarChange(null);
 
     const expectedActions = [
```

Comment:
Maybe here it is better to mock specifically fetchAuthorPublications and fetchAuthorPublicationsFacets and check just that they are called. The actions created are already tested in the authors.test.js. Also, we should unclear the mocks.

Reply:
There is no mock to clear here, as of now. Not sure what you mean by that. 

Also I think there was a reason (wasn't working) why we used directly `action` rather than mocking the action creators like we do almost everywhere else, that's why I kept it like that. But it may not be the case anymore I will to see if that works.

Reply:
I meant clear the mocks of the fetchAuthorPublications and fetchAuthorPublicationsFacets if we decide to go this way